test/unit/test_mechcat.cpp

```diff
@@ -217,11 +218,11 @@ mechanism_catalogue build_fake_catalogue() {
 
     // Add derived versions with global overrides:
 
-    cat.derive("fleeb1",        "fleeb",         {{"plugh", 1.0}}, {{"a", "b"}, {"b", "a"}});
-    cat.derive("special_fleeb", "fleeb",         {{"plugh", 2.0}});
-    cat.derive("fleeb2",        "special_fleeb", {{"norf", 11.0}});
-    cat.derive("fleeb3",        "fleeb1",        {}, {{"b", "c"}, {"c", "b"}});
-    cat.derive("bleeble",       "burble",        {{"quux",  10.}, {"xyzzy", -20.}});
+    cat.derive("fleeb1", "fleeb", {{"plugh", 1.0}}, {{"a", "b"}, {"b", "a"}});
+    cat.derive("special_fleeb", "fleeb", {{"plugh", 2.0}});
+    cat.derive("fleeb2", "special_fleeb", {{"norf", 11.0}});
+    cat.derive("fleeb3", "fleeb1", {}, {{"b", "c"}, {"c", "b"}});
+    cat.derive("bleeble", "burble", {{"quux", 10.}, {"xyzzy", -20.}});
 
```

Comment:
I think the original formatting is to be preferred here.

Reply:
You were too fast, I think that has been fixed?!